subprojects/dependency-management/dependency-management.gradle.kts

```diff
@@ -0,0 +1,63 @@
+import org.gradle.build.ClasspathManifest
+import org.gradle.gradlebuild.testing.integrationtests.cleanup.EmptyDirectoryCheck
+import org.gradle.gradlebuild.unittestandcompile.ModuleType
+
+plugins {
+    `java-library`
+}
+
+java {
+    sourceCompatibility = JavaVersion.VERSION_1_6
```

Comment:
❓This wasn't set in the Groovy script, why the change?

Reply:
Damn. Another merging issue. This is now covered by `moduleType = ModuleType.WORKER`. Thanks for the thorough check.